easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1-foss-2018b.eb

```diff
@@ -0,0 +1,74 @@
+# easybuild easyconfig
+#
+# Graphviz has moved to gitlab and it is no longer possible to get tagged
+# versions.  There is an open ticket this issue.
+
+easyblock = 'ConfigureMake'
+
+name = 'Graphviz'
+version = '2.40.1'
+
+homepage = 'http://www.graphviz.org/'
+description = """Graphviz is open source graph visualization software. Graph
+ visualization is a way of representing structural information as diagrams of
+ abstract graphs and networks. It has important applications in networking,
+ bioinformatics,  software engineering, database and web design, machine
+ learning, and in visual interfaces for other technical domains."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://graphviz.gitlab.io/pub/graphviz/stable/SOURCES/']
+sources = ['graphviz.tar.gz']
```

Comment:
Please use the alternate source URL we use in the existing easyconfig for `Graphviz` 2.40.1, see https://github.com/easybuilders/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1-intel-2018a.eb#L15

Reply:
source_urls changed
source_urls = ['https://fossies.org/linux/misc/']